.gitignore

```diff
@@ -1,4 +1,4 @@
 .bundle
 bin/configlet
 bin/configlet.exe
-Gemfile.lock
+/Gemfile.lock
```

Comment:
This tells git to ignore the `Gemfile.lock` at the repo root, so it will work (in fact is preferred).